lighthouse-core/scripts/git3po-rules/cannot-attach-to-this-target.yaml

```
@@ -0,0 +1,25 @@
+repo: GoogleChrome/lighthouse
+startAt: '2020-01-12T16:00:00Z'
+filters:
+  - type: issue
+    criteria:
+      text:
+        $or:
+          - $match: 'Extension Error: Cannot attach to this target'
+      state: open
+  - type: comments
+    length:
+      $lt: 2
+actions:
+  - type: add_comment
+    body: 'Howdy chief! Appreciate you filing this bug. :clap:
```

```suggestion
    body: 'Howdy! Appreciate you filing this bug. :clap:
```

I think we should remove "chief" from all of these.

---

i <3 "chief"